environment/src/main/java/com/sequenceiq/environment/environment/validation/network/azure/AzureExistingPrivateDnsZonesService.java

```diff
@@ -8,25 +8,45 @@
 
 import org.springframework.stereotype.Service;
 
+import com.sequenceiq.cloudbreak.cloud.azure.AzurePrivateDnsZoneDescriptor;
+import com.sequenceiq.cloudbreak.cloud.azure.AzurePrivateDnsZoneRegistrationEnum;
 import com.sequenceiq.cloudbreak.cloud.azure.AzurePrivateDnsZoneServiceEnum;
 import com.sequenceiq.environment.network.dto.NetworkDto;
 
 @Service
 public class AzureExistingPrivateDnsZonesService {
 
-    public Map<AzurePrivateDnsZoneServiceEnum, String> getExistingZones(NetworkDto networkDto) {
+    public Map<AzurePrivateDnsZoneServiceEnum, String> getExistingServiceZones(NetworkDto networkDto) {
         Map<AzurePrivateDnsZoneServiceEnum, String> result = new HashMap<>();
         Optional.ofNullable(networkDto.getAzure().getDatabasePrivateDnsZoneId())
                 .ifPresent(privateDnsZone -> result.put(AzurePrivateDnsZoneServiceEnum.POSTGRES, privateDnsZone));
         return result;
     }
 
+    public Map<AzurePrivateDnsZoneDescriptor, String> getExistingServiceZonesAsDescriptors(NetworkDto networkDto) {
+        Map<AzurePrivateDnsZoneDescriptor, String> result = new HashMap<>();
```

Comment:
some log would be nice

Reply:
I now added logs to methods getExistingServiceZones and getExistingRegisteredOnlyZonesAsDescriptors.